packages/ag-charts-enterprise/src/axes/angle-number/angleCrossLine.ts

```diff
@@ -0,0 +1,173 @@
+import { _ModuleSupport, _Scale, _Scene, _Util } from 'ag-charts-community';
+import { PolarCrossLine } from '../polarCrossLine';
+
+const { ChartAxisDirection } = _ModuleSupport;
+const { Path, Sector, Text } = _Scene;
+const { normalizeAngle360, isNumberEqual } = _Util;
+export class AngleCrossLine extends PolarCrossLine {
```

Comment:
This seems to be an identical implementation to the `angle-category` axis implementation? Can we factor out the common code please?